src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java

```diff
@@ -124,11 +124,7 @@ public boolean checkIfInstanceAlreadyExist(JsonObject instance) {
 
     public void writeIntoFile() {
         try {
```

Comment:
Ericssons copyright header seems to be missing in this file.

Reply:
No. Header was added by Andriy befor. So it exists here.

Reply:
Ok. But its no visible here. But if its there, then its ok.